backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/output/database/service/NormDBService.java

```diff
@@ -97,8 +95,7 @@ public Optional<Norm> loadNormByGuid(LoadNormByGuidPort.Options options) {
       .map(NormManifestationMapper::mapToDomain);
   }
 
-  @Override
-  public Optional<Norm> updateNorm(UpdateNormPort.Options options) {
+  private Optional<Norm> updateNorm(UpdateOrSaveNormPort.Options options) {
```

Comment:
So you removed UpdateNormPort and converted the overridden method to a private function. Fine. But Why passing the options of the other port? Why not just take the Norm object as the input parameter? It doesn't feel right to use the input parameter of the port, specially if the options only have a norm object